config/packages/doctrine.yaml

```diff
@@ -3,7 +3,7 @@ doctrine:
         # configure these for your database server
         driver: 'pdo_pgsql'
         charset: utf8
-        server_version: 9.6
+        server_version: 12
```

Comment:
We probably should leave this at 9.6 as is checked in in code (stash it after making the migration), or will that make the migration not compatible?

Reply:
Tried it, migration works at 9.6. But we should specify the correct version of postgres in the future